conda/utils.py

```diff
@@ -389,3 +389,11 @@ def exp_backoff_fn(fn, *args):
                 raise
         else:
             return result
+
+# put back because of conda build
+def url_path(path):
```

Comment:
this can be

    from .common.url import path_to_url
    urlpath = path_to_url